scio-google-cloud-platform/src/main/scala/com/spotify/scio/bigquery/MockBigQuery.scala

```diff
@@ -31,11 +31,13 @@ import scala.reflect.runtime.universe._
 /** Companion object for [[MockBigQuery]]. */
 object MockBigQuery {
 
-  /** Create a new MockBigQuery instance. */
-  def apply(): MockBigQuery = new MockBigQuery(BigQuery.defaultInstance())
-
   /** Create a new MockBigQuery instance with the given BigQueryClient. */
-  def apply(bq: BigQuery): MockBigQuery = new MockBigQuery(bq)
+  def apply(
+    bq: BigQuery = BigQuery.defaultInstance(),
+    stagingProject: Option[String] = None,
+    stagingDataset: Option[String] = None
+  ): MockBigQuery =
+    new MockBigQuery(bq = bq, stagingProject = stagingProject, stagingDataset = stagingDataset)
```

Comment:
```suggestion
    new MockBigQuery(bq, stagingProject, stagingDataset)
```

just a suggestion to simplify bc i don't think the shadowed field names are needed for clarity.